downloader/src/main.rs

```diff
@@ -60,11 +63,16 @@ async fn process_episode(
     })
 }
 
+/// Download remote resource without allocating too much memory
```

Comment:
I don't think that allocating memory for an mp3 file is a problem when you already hold multiple AI pytorch models (around 5G each) but this is cool, thanks :+1:

Reply:
Depends on how much RAM you have or wants to spend cash on cloud I would argue this is exactly why you should preserves as much RAM intact as possible. You are losing 5G at start.

Also loading everything to RAM and then writing to fs will be considerably slower

Reply:
Agreed, that's why I think the change is much appreciated :+1:

As for the amount of RAM for this batch job - I think it will run on something with 256 - 512GB of RAM ;)